### PR TITLE
Update migration guide 2.4 about Anorm large parameters

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

**Comment by marcospereira** [comment]
@play-pull-request-validator

**Comment by cchantep** [comment]
Scalariform SBT plugin issue

**Comment by marcospereira** [comment]
@cchantep there is a lot of other changes here. Was that caused by scaliform upgrade? If so, maybe we should split this in two separated PRs. 

What do you think?

File: framework/project/plugins.sbt [hunk 1]
```diff
@@ -34,7 +34,7 @@ libraryDependencies ++= Seq(
 
 // override scalariform version to get some fixes
 
-libraryDependencies += "org.scalariform" %% "scalariform" % "0.1.5-20140822-69e2e30"
+libraryDependencies += "org.scalariform" %% "scalariform" % "0.1.6"
```
Comment (jroper) [inline framework/project/plugins.sbt]: Let's not upgrade this in this PR, keep it separate.

Comment (cchantep) [inline framework/project/plugins.sbt]: Just try to fix the SBT issue which is blocking.

**Pull request merged**